packages/core/src/conflict-resolution.ts

```diff
@@ -271,24 +267,21 @@ export class ConflictResolution {
     const itr = log.entries()
     let entry = itr.next()
     while (!entry.done) {
-      const logEntry = entry.value[1]
-      const commit = await this.dispatcher.retrieveCommit(logEntry.cid)
+      const logEntry = await this.expandLogEntry(entry.value[1])
```

Comment:
why do we need this call?  Won't this have already been done by `fetchLog`?

Reply:
I did it this way because there were calls being made to `applyLogToState()` via  `snapshotAtCommit()` _without_ a prior call to `fetchLog()`. This seemed like a good reason to just do it everywhere. If the commit had already been unwrapped, the extra function call would just be a no-op.

Besides, now we have one place where commits are unwrapped that we can update/optimize in the future.

Reply:
okay fair enough, makes sense